CMakeLists.txt

```diff
@@ -121,6 +121,8 @@ if (TG_OWT_USE_PIPEWIRE)
         link_pipewire(tg_owt)
     endif()
 
+    link_glib(tg_owt)
+
```

Comment:
How is linking glib related to building with(out) pipewire?

Reply:
glib is used only by pipewire-related code for dbus communication

Reply:
Ah, it moved into the `if (TG_OWT_USE_PIPEWIRE)` block, I missed that.